pkg/diffs/diffs.go

```diff
@@ -105,7 +106,7 @@ func GetChangedPresubmits(prowMasterConfig, prowPRConfig *prowconfig.Config, log
 			masterJob := masterJobs[repo][job.Name]
 			logFields := logrus.Fields{logRepo: repo, logJobName: job.Name}
 
-			if job.Agent == string(pjapi.KubernetesAgent) {
+			if job.Agent == string(pjapi.KubernetesAgent) && !job.Hidden {
```

Comment:
I don't think the check belongs here. We have the two methods (`filterPresubmits` and `filterPeriodics`) whose job is to remove the presubmits that should be rehearsed but for some reason can't be, that seems like a much better place.